### ISSUE TITLE
🐛 missing `license` on optional dependencies

### ISSUE THREAD

**Issue by jorgebodega** [rome/tools#4290]
### Environment information

```block
No information, as is not a bug in rome itself
```

### What happened?

I've been struggling with a license checker and just see that generated `cli` packages don't have license on npmjs, [here an example](https://www.npmjs.com/package/@rometools/cli-linux-arm64/v/11.0.0).

As far as I can see, should be easy to add it, just want to ask if is just missing or if is intended.

### Expected result

The `cli` dependencies of `rome` package should have a valid license.

### Code of Conduct

- [X] I agree to follow Rome's Code of Conduct

**Comment by jorgebodega** [comment]
If is only missing, I can send a PR.

**Comment by ematipico** [comment]
Actually, these packages are created dynamically, this is the script: https://github.com/rome/tools/blob/main/npm/rome/scripts/generate-packages.mjs

We run it in our CI. Is adding the license field enough?

**Comment by jorgebodega** [comment]
Yes, that is exactly what I saw. And yes, is enough.

I suppose on how you speak that is just something forgotten, I can send a PR if that helps.

**Comment by ematipico** [comment]
Yes, it was an overnight. Feel free to send a PR, that would be amazing!
